src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -551,14 +551,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
             Dim Here = 1
             Dim atNewLine As Boolean
+            ' Line continuation is valid at the end of the line, or at the end of the file, or followed by a trailing comment.
+            ' Eg.  LineContinuation ( EndOfLine | EndOfFile | LineContinuationComment)
+            ' PROTOTYPE(continuation-comments): update once LangVersion 16 is created
+            ' TODO the following line Feature.NonTrailingNamedArguments needs to change to be the correct feature name
```

Comment:
This PROTOTYPE and TODO can be removed, since nothing to touch in this file to correct for VB16. Thanks